pkg/preference/preference.go

```diff
@@ -117,6 +117,15 @@ type OdoSettings struct {
 
 	// PushTarget for telling odo which platform to push to (either kube or docker)
 	PushTarget *string `yaml:"PushTarget,omitempty"`
+
+	// RegistryList for telling odo to connect to all the registries in the registry list
+	RegistryList *[]Registry `yaml:"RegistryList,omitempty"`
+}
+
+// Registry includes the registry metadata
+type Registry struct {
```

Comment:
Thinking out loud here. Does it add value or make sense to name this as `DevfileRegistry`? Just in case we decide to support registry of something else in the future? Or if devfile v2 might have a registry that might need a different struct of itself? WDYT @kadel @elsony

Reply:
If we do so, it will percolate to various other places in the code like `RegistryList`.

Reply:
Registry is the fundamental struct for any kind of registry as it only includes `Name` and `URL` fields, for future support we can use "inheritance", I know Go doesn't support "inheritance" but we still can use "composition" to achieve that.

Reply:
Yep. Agree. :smile: